### ISSUE TITLE
spdk_top crashed while adding nvme bdev

### ISSUE THREAD

**Issue by KariukiJ** [spdk/spdk#1436]
spdk_top application crashed while adding nvme bdev to nvmf_tgt.
I started the nvmf_tgt using the following command
app/nvmf_tgt/nvmf_tgt -m 0x3 &

Created TCP transport and then launched the spdk_top application. I looked around spdk_top checking out all the information on different screens and then went to the Poller screen 

I added an nvme bdev using command:
scripts/rpc.py bdev_nvme_attach_controller -b Nvme0 -t PCIe -a 0000:60:00.0
spdk_top crashed with a Seg fault
 
## Expected Behavior

I expected to see a bdev poller added to the pollers screen and then the spdk_top application starts showing the metrics for that poller 

## Current Behavior

A bdev_nvme_poll_adminq poller was added to the poller screen and then the application crashed with a seg fault

## Steps to Reproduce

1. started the nvmf_tgt using the following command
app/nvmf_tgt/nvmf_tgt -m 0x3 &
2. Created TCP transport and then launched the spdk_top application
3. Add an nvme bdev using command:
scripts/rpc.py bdev_nvme_attach_controller -b Nvme0 -t PCIe -a 0000:60:00.0

## Context (Environment including OS version, SPDK version, etc.)

Linux fedora27-SPDK.localdomain 5.7.0
SPDK commit da6581ade608e
--

--

**Comment by KariukiJ** [comment]
When I restarted the spdk_top application, I didn't see the crash when I added another nvme dev. However, when I restart the nvmf_tgt and start the spdk_top application, I reproduced the issue when I added the first nvme bdev while spdk_top is running. So it seems that the crash is happening for the first nvme bdev but once I restart the spdk_top I can add bdevs without any issues. 
I have the core dump but cannot attach it to git because it in lzo format and git does not support lzo.
